chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/MeaningfulUseAccurateAsOf.java

```diff
@@ -0,0 +1,97 @@
+package gov.healthit.chpl.domain;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+
+import gov.healthit.chpl.dto.MeaningfulUseAccurateAsOfDTO;
+
+public class MeaningfulUseAccurateAsOf implements Serializable {
+	private static final long serialVersionUID = -4803763243075068608L;
+	
+	private Long id;
+	private Long accurateAsOfDate;
+	private Boolean deleted;
+	private Long lastModifiedUser;
+	private Long creationDate;
+	private Long lastModifiedDate;
+	
+	private Set<String> warningMessages = new HashSet<String>();
```

Comment:
I'm adding an error in the controller when the POST update is called without an accurateAsOfDate. I added the warningMessages in as convention since that seems to be the standard in other objects returned by the API.